src/test/java/io/vertx/ext/mongo/AggregateOptionsTest.java

```diff
@@ -0,0 +1,77 @@
+package io.vertx.ext.mongo;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.test.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author <a href="mailto:nscavell@redhat.com">Nick Scavelli</a>
+ */
+public class AggregateOptionsTest {
```

Comment:
I think you're missing an explicit test to test the tmp disk write.

Reply:
added